README.md

```diff
@@ -75,50 +115,26 @@ In addition to our own documentation, here are some additional, relevant article
 
 ## Community and Feedback
 
-The ML-Agents toolkit is an open-source project and we encourage and welcome
+The ML-Agents Toolkit is an open-source project and we encourage and welcome
 contributions. If you wish to contribute, be sure to review our
 [contribution guidelines](com.unity.ml-agents/CONTRIBUTING.md) and
 [code of conduct](CODE_OF_CONDUCT.md).
 
-For problems with the installation and setup of the the ML-Agents toolkit, or
+For problems with the installation and setup of the the ML-Agents Toolkit, or
 discussions about how to best setup or train your agents, please create a new
 thread on the [Unity ML-Agents forum](https://forum.unity.com/forums/ml-agents.453/)
 and make sure to include as much detail as possible.
-If you run into any other problems using the ML-Agents toolkit, or have a specific
+If you run into any other problems using the ML-Agents Toolkit, or have a specific
 feature requests, please [submit a GitHub issue](https://github.com/Unity-Technologies/ml-agents/issues).
 
-Your opinion matters a great deal to us. Only by hearing your thoughts on the Unity ML-Agents Toolkit can we continue
-to improve and grow. Please take a few minutes to [let us know about it](https://github.com/Unity-Technologies/ml-agents/issues/1454).
+Your opinion matters a great deal to us. Only by hearing your thoughts on the Unity ML-Agents
+Toolkit can we continue to improve and grow. Please take a few minutes to
+[let us know about it](https://github.com/Unity-Technologies/ml-agents/issues/1454).
 
 For any other questions or feedback, connect directly with the ML-Agents
```

Comment:
Do we still want to recommend direct email?

Reply:
I think for confidential info sharing from developers or researchers it makes sense for there to be a way for them to contact us non-publicly.

Reply:
Yeah, I think we should.